app/code/Magento/Paypal/Model/Express/Checkout.php

```diff
@@ -809,7 +809,11 @@ public function place($token, $shippingMethodCode = null)
             case \Magento\Sales\Model\Order::STATE_PROCESSING:
             case \Magento\Sales\Model\Order::STATE_COMPLETE:
             case \Magento\Sales\Model\Order::STATE_PAYMENT_REVIEW:
-                $this->orderSender->send($order);
+                try {
+                    $this->orderSender->send($order);
```

Comment:
Code changes look good to me :+1: 

Please cover new logic with unit test, make an `orderSender` mock so that `send` is throwing exception and then assert exception was logged.

Reply:
@orlangur I tried to start doing this but it looks like it is not as simple as I thought. No tests exist for the placeOrder at this time so I'd need a quote management mock, an order mock as it returns an order, and then the order sender mock, and I'm not willing to commit so much time to learn all that and learn how to run individual tests.

If you can provide an existing test for placeOrder I'd likely be able to commit the time to build on it. But at the moment it's not something I'd be willing to proceed with further.